adminSiteClient/EditorMarimekkoTab.tsx

```diff
@@ -83,9 +87,25 @@ export class EditorMarimekkoTab extends React.Component<{ grapher: Grapher }> {
                                     value || undefined)
                         )}
                     />
+                    <Toggle
+                        label="Start with empty selection and explicitly add entities below"
+                        value={invertExcludedEntitiesList}
+                        onValue={action(
+                            (value: boolean) =>
+                                (grapher.invertExcludedEntitiesList =
+                                    value || undefined)
+                        )}
+                    />
                     <SelectField
-                        label="Exclude individual entities"
-                        placeholder="Select an entity to exclude"
+                        label={
+                            invertExcludedEntitiesList
+                                ? "Include individual entities"
+                                : "Exclude individual entities"
+                        }
+                        placeholder={
+                            "Select an entity to " +
+                            (invertExcludedEntitiesList ? "include" : "exclude")
+                        }
```

Comment:
This is great and it's very likely useful for other charts. Should we extract it as a reusable component/transform? Marcel [suggested this in the previous PR](https://github.com/owid/owid-grapher/pull/965#discussion_r676579614), and I think it's a good idea if there is no rush to merge this. If there is a rush to launch, then it's fine to leave it as it is. 

Regardless, I would be happy to implement it next week, I have some time. I [checked the database](https://github.com/owid/owid-grapher/pull/965#discussion_r677195472) and only ScatterPlot and Marimekko had `excludedEntities` populated, so it's safe to enable for all charts. We could move it to the Customize tab and have it available for all chart types.

Reply:
Good idea - I created a task to tackle this separately so we can publish marimekko without this: https://www.notion.so/owid/Add-include-exclude-entities-to-all-charts-724e198dfab54d4cb3512fed1dda82bc